### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by boredhero** [wacossusca34/glava#248]
Noticed that this project seems to be dead. I use it, but at this point, it is segfaulting. Does anyone know if it has been forked anywhere, or if there is a nexus of active development? If not, I may fork it and try to get it working.

**Comment by PixelMelt** [comment]
the fork at https://gitlab.com/wild-turtles-publicly-release/glava/glava still works but its not in development

**Comment by HumanG33k** [comment]
I can find time to improve the software if needed.

**Comment by LyrinoxTechnologies** [comment]
I've actually revived GLava.

https://git.pixelridgesoftworks.com/PixelRidge-Softworks/glava-ridged

**Comment by mscheper** [comment]
> I've actually revived GLava.

@VetheonGames : Cool. I didn't find any installation/build instructions, though. If they exist, could you put them in README.md? And will there be a PPA?

**Comment by LyrinoxTechnologies** [comment]
> > I've actually revived GLava.
> 
> @VetheonGames : Cool. I didn't find any installation/build instructions, though. If they exist, could you put them in README.md? And will there be a PPA?

Absolutely. I'll whip up some more documentation for it today. But the existing build instructions still work the exact same. There's not been any big changes or anything yet.

As for a PPA, there is a working PKGBUILD available for it on the AUR if you run Arch, but I've not got a flatpack or .Deb or anything else like that quite yet.

I can spin up a little APT repo for it, and build a .Deb later today and maybe an RPM one after.

**Comment by LyrinoxTechnologies** [comment]
@mscheper full disclosure, I've never built a .deb before.

However, I have finished some semi-decent documentation here: https://git.pixelridgesoftworks.com/PixelRidge-Softworks/glava-ridged/wiki/Home

Edit: It took a lot of doing, but I've successfully got RPM and DEB packages built, spun up a RPM and APT repo server, and wrote a bunch of docs with install instructions. I hope that helps out, and you guys can get back to making visualizers!

I'll be making a system soon for people to use to distribute their custom visualizers.

**Comment by isshin1** [comment]
hey @VetheonGames i am unable to access the link, seems like the domain is dead. Any plans of it coming back up ?

**Comment by tockudex** [comment]
Is anyone interested in actively forking this project, as in trying to maintain it?
Because previous attempts never lead anywhere...

**Comment by TornaxO7** [comment]
(Shameless self promotion)
I wrote something similar to glava (inspired from it): https://github.com/TornaxO7/vibe

and I'm open for some feedback ^^

**Comment by hero-hz1999yt** [comment]
> [@mscheper](https://github.com/mscheper) full disclosure, I've never built a .deb before.
> 
> However, I have finished some semi-decent documentation here: https://git.pixelridgesoftworks.com/PixelRidge-Softworks/glava-ridged/wiki/Home
> 
> Edit: It took a lot of doing, but I've successfully got RPM and DEB packages built, spun up a RPM and APT repo server, and wrote a bunch of docs with install instructions. I hope that helps out, and you guys can get back to making visualizers!
> 
> I'll be making a system soon for people to use to distribute their custom visualizers.

Your project is also dead
